src/worker/typosquat/config.rs

```diff
@@ -0,0 +1,65 @@
+//! Things that should really be read from a configuration file, but are just hardcoded while we
+//! experiment with this.
+
+use std::time::Duration;
+
+/// How long to cache the top crates for.
+pub(super) static CACHE_TTL: Duration = Duration::from_secs(12 * 60 * 60);
+
+/// Valid characters in crate names.
+pub(super) static CRATE_NAME_ALPHABET: &str =
+    "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz1234567890-_";
+
+/// E-mail addresses to notify when a potential typosquat is found.
+pub(super) static NOTIFY_EMAILS: &[&str] = &[
+    "adamharvey@rustfoundation.org",
+    "walterpearce@rustfoundation.org",
+];
+
+/// The number of crates to consider in the "top crates" corpus.
+pub(super) static TOP_CRATES: i64 = 3000;
+
+/// This is based on a pre-existing list we've used with crates.io for "easily confused
+/// characters". (I'm not really sure that I consider all of these easily confused, but it's better
+/// than nothing.)
```

Comment:
I assume this is based on https://github.com/dangardner/typogard/blob/5ce51b2ec45b7d4847dd944bc3048b206840f873/typogard_crates.py#L80-L123?

The "based on QWERTY keyboard locality" part of the original comment might be relevant here. I'm wondering if we should add QWERTZ and AZERTY locality too? QWERTZ should be relatively straight-forward since it's only a single character that was switched. AZERTY might increase the size of the matrix a bit more though.

Reply:
Good catch. Yeah, adding QWERTZ and AZERTY wasn't hard, so I did that.